package.json

```diff
@@ -74,6 +74,7 @@
     "benchmark": "^2.1.0",
     "common-tags": "^1.3.0",
     "coveralls": "^2.11.9",
+    "dedent": "^0.6.0",
```

Comment:
Do we want to use this separate package or [`stripIndent`](https://github.com/declandewet/common-tags#stripindent) from the already included `common-tags`?